src/main/events/helpers/generate-lutris-yaml.ts

```diff
@@ -1,9 +1,15 @@
 import { Document as YMLDocument } from "yaml";
 import { Game } from "@main/entity";
 import path from "node:path";
+import fs from "node:fs";
 
 export const generateYML = (game: Game) => {
   const slugifiedGameTitle = game.title.replace(/\s/g, "-").toLocaleLowerCase();
+  const gamePath = path.join(game.downloadPath, game.folderName);
+
+  const files = fs.readdirSync(gamePath);
```

Comment:
Please don't use synchronous operations when using I/O since this might potentially lock the main thread